CHANGELOG.md

```diff
@@ -8,6 +8,15 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 - `package-lock.json` for `npm5` users
+- Forced update to the newest resin-sdk (^6.4.1)
+- Allow OS version selection when doing `resin device init`
+- Actually tolerate the `--yes` param to `resin device init`
+- Allows passing `--drive` to `resin device init`
+- List detected drives with `resin os available-drives`
```

Comment:
I'm very positive about including it. It's a nice thing to have (because of the system drives filtering), and we already have this functionality. Yes it's not related to the OS per se, but it's the best NS we have as it's still related to `os initialize`